src/discussions/comments/reply/Reply.jsx

```diff
@@ -21,7 +21,7 @@ function ReplyHeader({ reply, intl }) {
         <div className="status small">
           <a href="#nowhere">
             <h1 className="font-weight-normal text-info-300 mr-1 small">
-              {reply.author}
+              {reply.author} {reply.authorLabel ? `[${reply.authorLabel}]` : '' }
```

Comment:
This is a shorter representation. Also I've updated it to use the correct brackets.
```suggestion
              {reply.author} {reply.authorLabel  && `(${reply.authorLabel})`}
```

Reply:
Oh! It looked like a square bracket in the mockup